### ISSUE TITLE
Use of unresolved identifier 'spring'

### ISSUE THREAD

**Issue by sliu410** [MengTo/Spring#140]
I drop the spring folder into my Xcode project and I am able to play with some autostart animation. However, when I wrote "spring(0.5){}" in the .swift file I got the bug in the title. Does anyone know how to solve it or can point me to any tutorial? Thanks!

**Comment by sahasd** [comment]
Can you be a bit more elaborate on the problem, perhaps a screenshot of where the error is occurring or the snippet of code with its file name?

**Comment by hamishirving** [comment]
I have the same issue, not recognising `spring` as a function?

Xcode 7.2 using latest Spring library

![designcode](https://cloud.githubusercontent.com/assets/2860879/13400136/557d9d16-defe-11e5-9608-5c8b68d76a6a.png)

**Comment by parane** [comment]
yep I also has same problem  when updating to swift 2 .... any fix ??

**Comment by parane** [comment]
SpringAnimation.spring(0.5) upgrade swift 2
